docs/source/classroom/class_guide.md

```diff
@@ -0,0 +1,172 @@
+
+
+
+::::{list-table}
+---
+header-rows: 1
+---
+* - Name
+  - Time Limit (default/max)
+  - Running Jobs (max)
+  - RAM Limit
+* - courses
+  - 4 hr / 24 hrs
+  - 50
+  - 256 GB
+* - courses-gpu
+  - 4 hr / 8 hrs
+  - 8
+  - 12 GB
+::::
+
+
+The resources available in the courses/courses-gpu partitions can be queried with the command `sinfo` as run in the command line. We manage the resourses in courses/courses-gpu each term in response to the number of courses and requested usage per course.
+
+:::{code-block} bash
+sinfo -p courses-gpu  --Format=nodes,cpus,gres,statecompact
+:::
+
+
+These partitions can be used in the following ways:
+
+(sbatch-courses-index)=
+### sbatch script
+
+An sbatch script can be submitted on the command line via the command `sbatch scriptname.sh`. Below are some examples of sbatch scripts using the courses and courses-gpu partitions. See {ref}`slurm-running-jobs` for more information on running sbatch scripts or run `man sbatch` for additional sbatch parameters.
+
+::::{dropdown} courses partition
+:::{code-block}bash
+
+#!/bin/bash
+
+#SBATCH --nodes=1  
+#SBATCH --time=4:00:00  
+#SBATCH --job-name=MyJobName  
+#SBATCH --partition=courses  
+#SBATCH --mail-type=ALL  
+#SBATCH --mail-users=$USER@northeastern.edu  
+
+#commands to execute  
+
+:::
+::::
+
+
+::::{dropdown} courses-gpu partition
+:::{code-block}bash
+
+#!/bin/bash
+
+#SBATCH --nodes=1  
+#SBATCH --time=4:00:00  
+#SBATCH --job-name=MyJobName  
+#SBATCH --partition=courses-gpu  
+#SBATCH --gres=gpu:1  
+#SBATCH --mail-type=ALL  
+#SBATCH --mail-users=$USER@northeastern.edu  
+
+#commands to execute  
+
+:::
+::::
+
+(srun-courses-index)=
+### srun interactive session
+
+An interactive session can be run on the command line via the `srun` command as shown in the examples below. See {ref}`slurm-running-jobs` for more information on using `srun` or run `man srun` to see additinal parameters that can be set with `srun`.
+
+
+::::{dropdown} courses partition
+:::{code-block} bash
+srun --time=4:00:00 --job-name=MyJob --partition=courses --pty /bin/bash
+:::
+::::
+
+::::{dropdown} courses-gpu partition
+:::{code-block} bash
+srun --time=4:00:00 --job-name=MyJob --partition=courses-gpu --gres=gpu:1 --pty /bin/bash
+:::
+::::
+
+(OOD-courses-index)=
+### Open OnDemand
+
+We have several widely-used applications available on the Open OnDemand (OOD) website including, Jupyterlab Notebook, Rstudio, Matlab, GaussView and more. {ref}`access-ood`
+
+All of the applications under the “Courses” tab on the dashboard can be set to either the `courses` or `courses-gpu` partitions via the applications specific pull down menus.
+
+### Monitoring Jobs
+
+Whichever way you choose to run your jobs, you can monitor their progress with the command `squeue`.
+
+:::{code-block} bash
+squeue -u username
+:::
+
+You can also monitor jobs being run on either of the courses partitions.
+
+:::{code-block} bash
+squeue -p courses
+squeue -p courses-gpu
+:::
```

Comment:
It might be interesting to have a section on checking running jobs on OOD. The 'My Interactive Session' tab and to remind students to cancel jobs/sections when they don't have anything running.